src/store/actions/mainForm/index.ts

```diff
@@ -0,0 +1,6 @@
+import { ADD_DATA } from "store/types";
+
+export const addUserInfoToReduxAction = (userInfo: any) => ({
+  type: ADD_DATA,
```

Comment:
also redux types names should be meaningful

Reply:
Name of this redux-type is changed